network-api/networkapi/buyersguide/models.py

```diff
@@ -0,0 +1,90 @@
+from django.db import models
+from networkapi.utility.images import get_image_upload_path
+# from wagtail.snippets.models import register_snippet
+
+
+def get_product_image_upload_path(instance, filename):
+    return get_image_upload_path(
+        app_name='buyersguide',
+        prop_name='name',
+        instance=instance,
+        current_filename=filename
+    )
+
+
+class Product(models.Model):
+    """
+    A thing you can buy in stores and our review of it
+    """
+
+    name = models.CharField(
+        max_length=100,
+        help_text='Name of Product',
+        blank="True",
+    )
+    company = models.CharField(
+        max_length=100,
+        help_text='Name of Company',
+        blank="True",
+    )
+
+    blurb = models.TextField(
+        max_length=5000,
+        help_text='Description of the product',
+        blank="True"
+    )
+    url = models.URLField(
+        max_length=2048,
+        help_text='Link to this product page',
+        blank="True",
+    )
+    price = models.CharField(
+        max_length=100,
+        help_text='Price range',
+        blank="True",
+    )
+    image = models.FileField(
+        max_length=2048,
+        help_text='Image representing this prodct',
+        upload_to=get_product_image_upload_path,
+        blank=True,
+    )
+    camera = models.NullBooleanField(
```

Comment:
why do we need true, false, or null?

Reply:
Because the answer to some of these is "unknown". This allows editors to leave the field unfilled.